### PR TITLE
Workaround/dev 430 app store fix (loans var a + b)

### PR DESCRIPTION
**Pull request by ThaBrad**
#### What kind of PR is this?:
/kind feature
/kind fix

#### Developer Checklist:
- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Loans/components/EmptyVault.tsx [hunk 1]
```diff
@@ -68,6 +69,14 @@ export function EmptyVault (props: EmptyVaultProps): JSX.Element {
           text='Learn more about vaults'
           testId='empty_vault_learn_more'
         />
+        <TouchableOpacity onPress={() => navigation.navigate('LoansTnC')}>
+          <InfoText
+            simple
+            text={translate('components/LoansFaq', 'Loan terms')}
+            style={tailwind('p-2 flex-row items-end justify-start')}
+          // testId='empty_vault_learn_more'
```
Comment (Krysh90) [inline mobile-app/app/screens/AppNavigator/screens/Loans/components/EmptyVault.tsx:77]: what is this testId? is it used for unit & integration testing? why is it commented out?

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnC.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        text: translate('components/LoansFaq', useTnC()),
+        type: 'paragraph'
+      }]
+    }
+  ]
+
+  return (
+    <ThemedScrollView
+      contentContainerStyle={tailwind('p-6 pb-8')}
+      testID='loans_faq'
```
Comment (Krysh90) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnC.tsx:25]: on `InfoText` there is a `testId`, why is it called here `testID`?

Comment (ThaBrad) [reply mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnC.tsx:25]: This is an identifier for tests (which we currently are not doing and not really planned as really a pain in the a** with merging and our current and future code changes).
I'll try to still keep em in, _just in case_:

File: mobile-app/app/screens/AppNavigator/screens/Loans/components/EmptyVault.tsx (hunk 1 above)
Comment (ThaBrad) [reply mobile-app/app/screens/AppNavigator/screens/Loans/components/EmptyVault.tsx:77]: most probably for both of 'em

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnCText.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+export function useTnC (): string {
+    const { language } = useLanguageContext()
+
+    const text = {
+        en: tncText,
+        de: tncTextDe
+    }
+
+    return text[language]
```
Comment (davidleomay) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnCText.tsx:12]: If my language is not english/german:
telegram-cloud-photo-size-4-5965426866967919209-y

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnC.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import { AccordionContent, WalletAccordion } from '@components/WalletAccordion'
+import { StackScreenProps } from '@react-navigation/stack'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import { LoanParamList } from '../LoansNavigator'
+import { useTnC } from './LoansTnCText'
+
+type Props = StackScreenProps<LoanParamList, 'LoansTnC'>
+
+export function LoansTnC ({ route }: Props): JSX.Element {
```
Comment (davidleomay) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnC.tsx:11]: Do we need this component, or could we just redirect the user to the LoansFaq?

Comment (ThaBrad) [reply mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnC.tsx:11]: better safe than sorry !

…also LoansFaq opens a different dropdown by default, explaining the the question that it routes from.

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnCText.tsx (hunk 3 above)
Comment (ThaBrad) [reply mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnCText.tsx]: extremely good catch!

**Pull request merged**